rdkit/Chem/UnitTestCatalog.py

```diff
@@ -138,33 +138,27 @@ def test4CatGuts(self):
               (15, 'CC=C<-O>C', 3, (34, )),
               (16, 'C=CC<-O>', 2, (34, )), ]
     for i in range(len(descrs)):
-      id, d, order, ids = descrs[i]
-      descr = self.fragCat.GetBitDescription(id)
-      assert descr == d, '%d: %s != %s' % (id, descr, d)
-      assert self.fragCat.GetBitOrder(id) == order
-      assert tuple(self.fragCat.GetBitFuncGroupIds(id)) == \
-             ids,'%d: %s != %s'%(id,
-                                 str(self.fragCat.GetBitFuncGroupIds(id)),
-                                 str(ids))
+      ID, d, order, ids = descrs[i]
+      descr = self.fragCat.GetBitDescription(ID)
+      assert descr == d, '%d: %s != %s' % (ID, descr, d)
```

Comment:
Agree. However in this case, there is sufficient effort in getting a useful error message if the assert fails. self.assertEqual wouldn't give anything else.